src/app/interfaces/catalogue-details.ts

```diff
@@ -0,0 +1,4 @@
+export interface CatalogueDetail {
```

Comment:
I wonder if this name is correct. What does it retrieve example? Field values and counter per value?